### DISCUSSION TITLE
导出查询页面崩溃咋办？

### DISCUSSION THREAD

**Discussion by a8159236** [hhyo/Archery#2792] [Q&A]
查询数据（总共 114969 条记录）导出时直接页面卡死，然后页面崩溃了，这个导出有大小限制吗？能设置吗不然数据一多页面就崩溃了。

![image](https://github.com/user-attachments/assets/150c5396-75f5-4d7b-8b17-1365632a32c5)
![企业微信截图_17255948992385](https://github.com/user-attachments/assets/2f956835-7074-408f-bbe2-3e4d41af8b1d)

**Comment by feiazifeiazi** [comment]
要等后端导出功能。

**Reply by a8159236** [reply]
好的谢谢
